doc/world_format.txt

```diff
@@ -145,6 +145,16 @@ For load_mod_<mod>, the possible values are:
     * Other locations and absolute paths are not supported
     * Note that `moddir` is the directory name, not the mod name specified in mod.conf.
 
+PostgreSQL backend specific settings:
+  pgsql<type>_connection = host=127.0.0.1 port=5432 user=mt_user password=mt_password dbname=minetest<type>
+  ^ where <type> is the backend type, i.e. "", "player", "readonly" or "auth"
```

Comment:
Documenting this separately would be clearer.

<pre>
pgsql_connection = host=127.0.0.1 port=5432 user=mt_user password=mt_password dbname=minetest
pgsql_player_connection = (same parameters as above)
pgsql_readonly_connection = (same parameters as above)
pgsql_auth_connection = (same parameters as above)
</pre>

Reply:
From what I can see, `dbname` changes depending on the backend, hence it's not the same parameters. How about `(same auth as above) dbname=minetestplayer`?

Reply:
Those are just example values, the parameters are totally independent. But you are correct that users should use different databases for them.